src/main/java/com/google/javascript/clutz/ImportRenameMapBuilder.java

```diff
@@ -49,10 +80,11 @@ private static boolean isGoogRequireAssignment(Node statement) {
 
     return rightHandSide != null
         && rightHandSide.isCall()
-        && rightHandSide.getFirstChild().matchesQualifiedName("goog.require");
+        && (rightHandSide.getFirstChild().matchesQualifiedName("goog.require")
```

Comment:
is goog.require in goog.scope even a thing? Just drop it for now. I haven' seen such a pattern in g3.

Reply:
This method matches either a `goog.require` or a `goog.module.get`.  Which it will find depends on whether a goog.module body was passed to `build` or a goog.scope body.  I've added comments.

Reply:
I see, 'goog.require' was there before. Makes sense.